docs/faq/network/blockchain-architecture.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: 'Blockchain Architecture'
+sidebar_position: 1
+---
+
+# Blockchain Architecture
+
+### What is LUKSOs Network Architecture?
+
+LUKSO's network architecture is based on the **Proof of Stake** consensus protocol, which relies on validators to secure the network and maintain consensus. The protocol is combined with two fundamental necessities to allow a secure and efficient network.
+
+The first necessity is **achieving the finality** of changes made to the network. This ensures that once a block is added to the blockchain, its information is immutable and cannot be changed. Finality is managed by the Casper the Friendly Finality Gadget (**Casper FFG**), a consensus mechanism designed to achieve this finality.
```

Comment:
"immutable and cannot be changed" -> it's the same thing
"Finality is managed by the Casper the Friendly Finality Gadget (**Casper FFG**)" -> remove 'the' just before Casper